benchmark/benchmarkPlanarGraphs.cpp

```diff
@@ -0,0 +1,31 @@
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include <cassert>
+#include <fstream>
+#include <io/G6GraphReader.hpp>
+#include <iostream>
+#include <map>
+#include <recognition/planar/PlanarGraphRecognition.hpp>
```

Comment:
Please split these headers into two groups: native C++ includes + NetworKit includes (see other benchmark files for reference)